custom-objects/type_adapters.md

```diff
@@ -53,4 +53,13 @@ class UserAdapter extends TypeAdapter<User> {
     writer.write(obj.name);
   }
 }
-```
\ No newline at end of file
+```
+
+## Check if Adapter is registered
+
+If you want to check if a type is already registered, you can use the `isAdapterRegistered` method and pass in the `typeId` as follows:
+```dart
+Hive.isAdapterRegistered(0);
+```
+
+This is useful say if you want to run `openBox` in multiple environments. Such an instance could be, you would like to run `openBox` in an isolate or background service and the app in the main isolate at the same time.
```

Comment:
We wouldn't suggest using hive in multi-isolate environment because dart isolates doesn't support shared memory so hive instances on multiple isolates wouldn't work as expected and might cause data loss (https://github.com/hivedb/hive/issues/263).